Source/AwsCommonRuntimeKit/crt/ByteBuffer.swift

```diff
@@ -71,8 +71,8 @@ public class ByteBuffer: Codable {
         return self
     }
 
-    public func put(_ value: ByteBuffer, offset: UInt = 0, maxBytes: UInt? = nil) {
-        var end: UInt = UInt(value.length)
+    public func put(_ value: ByteBuffer, offset: UInt = 0, maxBytes: UInt? = nil) throws {
+        var end: UInt = try UInt(value.length())
```

Comment:
don't make this function `throws`

ByteBuffer.length() cannot ever actually throw
the protocol API can throw
but the ByteBuffer's implementation never does
just do `!` that it succeeds

Reply:
Thanks, I found an interesting tidbit that we can satisfy a throwing function protocol requirement with a non throwing function.